README.md

```diff
@@ -2195,6 +2195,82 @@ container_image(name, base, data_path, directory, files, legacy_repository_namin
         </p>
       </td>
     </tr>
+    <tr>
+      <td><code>healthcheck_test</code></td>
+      <td>
+        <code>List of strings, optional</code>
+        <p><a href="https://docs.docker.com/engine/reference/builder/#healthcheck">
+        The test to perform to check that the container is healthy.</a></p>
+        <p>An empty list means to inherit the default.</p>
+        <p>Possible definition options are:</p>
+        <p><code>[]</code> : inherit healthcheck</p>
```

Comment:
`None` also inherits the healthcheck. It may be nice to specify this for callers who write their own macros wrapping this.

Reply:
Added. Thanks!